report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -208,13 +226,54 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
         }
       }
 
-      function toggleNextTest(currentTest) {
-        $(currentTest).nextAll('.aside-report:not(.is-hidden)')
-        .first()
-        .find('.test-name')
-        .click();
+      function toggleNextTestItem(currentTest) {
+        $(currentTest).nextAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .first()
+          .click();     
       }
 
+      function togglePrevTestItem(currentTest) {
+        $(currentTest).prevAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .last()
+          .click();
+      }
+
+      function clickOnTab() {
+        var currentTab = userSettingsService.getLastTab();
+        var nextTestTabs = $('.nav-tabs').children();
```

Comment:
Both lines - 248-249 - consider adding a `$` suffix if they store jQuery collections

Reply:
Here again currentTab is just plain text, will change that to currentTabLabel.
As to next line I will change that.